src/app/store/swap/epics.ts

```diff
@@ -11,18 +10,16 @@ import {
   Route3SwapParamsRequestRaw
 } from 'lib/apis/route3/fetch-route3-swap-params';
 import { fetchgetRoute3Tokens } from 'lib/apis/route3/fetch-route3-tokens';
+import { isDefined } from 'lib/utils/is-defined';
 
 import { loadSwapDexesAction, loadSwapParamsAction, loadSwapTokensAction, resetSwapParamsAction } from './actions';
 
 const isAmountDefined = (
   requestParams: Route3SwapParamsRequest | Route3SwapParamsRequestRaw
-): requestParams is Route3SwapParamsRequest => {
-  if (requestParams.amount !== undefined && requestParams.fromSymbol.length > 0 && requestParams.toSymbol.length > 0) {
-    return true;
-  }
-
-  return false;
-};
+): requestParams is Route3SwapParamsRequest =>
+  isDefined(requestParams.amount) !== undefined &&
```

Comment:
```suggestion
  isDefined(requestParams.amount)
```
Please check for eslint errors before applying the suggestion